server/api.proto

```diff
@@ -261,18 +261,26 @@ message Friend {
 }
 
 message TFriendAdd {
+  message UserIds {
```

Comment:
This means we can only add by IDs or by handles. How about using a message with a single `oneof` ID or handle, and repeating that? Does it make sense or would it be painful to handle server-side?